### PR TITLE
Replacing include guards with "#pragma once"

### LINKED ISSUE #2: Use `#pragma once`
Replace usage of:

```
#ifndef SOME_HEADER_FILENAME_H
#define SOME_HEADER_FILENAME_H
```

with 

```
#pragma once
```

This has been a standard technique for years now. And you will never forget that `#endif` at the end of the file :-D

### PR DESCRIPTION
**Pull request by wdhawkins**
Closes #2

### CODE REVIEW

File: framework/chi_configuration.h [hunk 1]
```diff
@@ -1,9 +1,7 @@
-#ifndef CHI_CONFIGURATION_H
-#define CHI_CONFIGURATION_H
+#pragma once
 
-#define PROJECT_VERSION "1.0.2"
+#define PROJECT_VERSION       "1.0.2"
```
Comment (ragusa) [inline framework/chi_configuration.h:3]: why the extra white space?

**Review by ragusa — APPROVED** [review]

File: framework/chi_configuration.h (hunk 1 above)
Comment (wdhawkins) [reply framework/chi_configuration.h]: Must be a side effect of my regular expression.

**Review by andrsd — CHANGES_REQUESTED** [review]
Remove the `framework/chi_configuration.h` file. It is generated by cmake from `resources/chi_configuration.h.in`

So, remove the second patch `git reset --hard HEAD~1`, remove the file, create a new patch and force-push the branch.

**Comment by wdhawkins** [comment]
> Remove the `framework/chi_configuration.h` file. It is generated by cmake from `resources/chi_configuration.h.in`
> 
> So, remove the second patch `git reset --hard HEAD~1`, remove the file, create a new patch and force-push the branch.

Thanks!  I didn't even notice that.  Fixed.

**Review by andrsd — APPROVED** [review]

**Pull request merged**